CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## 0.1.0
+  - Plugin created with the logstash plugin generator
```

Comment:
Updated changelog message. As this would be the initial release I simply stated what the plugin is/does.